src/main/java/hudson/plugins/git/util/GitUtils.java

```diff
@@ -357,34 +357,30 @@ public static EnvVars getPollEnvironment(AbstractProject p, FilePath ws, Launche
             }
         }
 
-        // add env contributing actions' values from last build to environment - fixes JENKINS-22009
-        addEnvironmentContributingActionsValues(env, b);
-
         EnvVars.resolve(env);
 
         return env;
     }
 
-    private static void addEnvironmentContributingActionsValues(EnvVars env, AbstractBuild b) {
-        List<? extends Action> buildActions = b.getAllActions();
-        if (buildActions != null) {
+    public static void addEnvironmentContributingActionsValues(EnvVars env, Job project, Run b) {
```

Comment:
Can you explain the change of data type from AbstractBuild to Run?  Is it because the lastBuild argument in GitSCM is a Run rather than an AbstractBuild?

Reply:
Correct, the lastBuild in a WorkflowJob is WorkflowRun which is instance of Run but not AbstractBuild.